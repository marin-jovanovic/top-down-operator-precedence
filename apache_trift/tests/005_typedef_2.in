include "temp"
include "t2"
include "t3"
