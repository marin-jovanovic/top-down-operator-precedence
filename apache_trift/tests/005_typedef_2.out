Document
 HeaderManager
  Header
   Include
    "include"
    Literal
     "temp"
  HeaderManager
   Header
    Include
     "include"
     Literal
      "t2"
   HeaderManager
    Header
     Include
      "include"
      Literal
       "t3"
    HeaderManager
     end of header instructions
 DefinitionManager
  end of definition instructions
